test/fixtures/BootstrapCommand/version-mismatch/packages/package-2/package.json

```diff
@@ -0,0 +1,8 @@
+{
+  "name": "package-2",
+  "version": "1.0.0",
+  "dependencies": {
+    "package-1": "^1.0.0",
```

Comment:
Yep, we're working with version independent projects which ideally should be correctly updated across the board, but it doesn't always happen.  Rather than have to enforce that with some other mechanism I was looking to just force linking with a flag like this.